### DISCUSSION TITLE
Como lidar com termos de múltiplos significados?

### DISCUSSION THREAD

**Discussion by levxyca** [levxyca/diciotech#188] [Q&A]
Uma de nossas pessoas contribuidores, levantou levantou uma questão importante que merece ser discutida com toda a comunidade do Diciotech! Queremos ouvir sua opinião e enriquecer essa discussão 🫱🏻‍🫲🏻

https://github.com/levxyca/diciotech/pull/151#issuecomment-2247949895

**Comment by FabianoRaiser** [comment]
Pode ser feito que nem a Wikipedia, onde quando há o mesmo titulo de artigo é endereçado as diferentes páginas. Por exemplo quando há dois ou mais eventos históricos com o mesmo nome, é posto um link para com uma página de desambiguação, no caso peguei o "Tratado de Paris" como [referencia](https://pt.wikipedia.org/wiki/Tratado_de_Paris).

**Reply by levxyca** [reply]
Muito bacana @FabianoRaiser! Acredito que complementa com a https://github.com/levxyca/diciotech/issues/12
